pkg/joinserver/redis/registry.go

```diff
@@ -206,7 +206,7 @@ func (r *DeviceRegistry) set(ctx context.Context, tx *redis.Tx, uid string, gets
 			pb = &ttnpb.EndDevice{}
 		}
 
-		pb.UpdatedAt = time.Now().UTC()
+		pb.UpdatedAt = ttnpb.ProtoTimePtr(time.Now().UTC())
```

Comment:
`UTC()` may be dropped. The reason is that `ProtoTime` converts to UTC on its own.